modules/ROOT/pages/managing-connected-salesforce-orgs.adoc

```diff
@@ -0,0 +1,104 @@
+
+When you remove the connection between your Anypoint Platform organization and Salesforce organization, the Salesforce organization no longer appears in the list of connected organizations. The connection still exists in the Salesforce organization until it's removed by the Salesforce organization administrator.
+
+To reestablish the connection between Anypoint Platform and removed Salesforce organizations, administrators with the appropriate permissions in both Anypoint Platform and Salesforce organizations must recreate the connection.
+
+[removing-connection-anypoint-platform]
+=== Remove a Connected Salesforce Org in Anypoint Platform
+
+An Anypoint Platform organization administrator can remove the connection between their Anypoint Platform organization and a connected Salesforce organization.
+
+When you remove a connection to a Salesforce organization, any capabilities that are associated with the Salesforce organization, such as API Catalog, are removed. 
+
+To remove the connection to a Salesforce organization:
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+. Click *Salesforce*.
+. Click the *Actions* menu (*...*) in the Salesforce organization to remove the connection from and select *Remove Salesforce Org*. 
+. In the *Remove Salesforce Org* dialog, click *Remove Org*.
+
+[removing-connection-salesforce]
+=== Remove a Connected Anypoint Platform Organization in Salesforce
+
+A Salesforce organization administrator can remove the connection between an Anypoint Platform organization and Salesforce organization using the Salesforce user interface. 
+
+To remove the connection to an Anypoint Platform organization in Salesforce:
+
+include::partial$include-salesforce-mulesoft-setup-nav.adoc[]
+
+. In the *Your connected Anypoint Platform Organization* section, click *View Connection Info*.
+. In the *Go To Anypoint* drop-down list, select *Remove connection*.
+. In the *Remove connection to Anypoint Platform* dialog box, click *OK*.
+
+Alternatively, if you want to temporarily prevent Anypoint Platform users from publishing assets to Salesforce from the Anypoint Platform side, you can instead <<enable-disable-connection,disable>> the connection temporarily.
+
+For more information about managing this relationship from the Salesforce user interface, see the https://help.salesforce.com/s/articleView?id=sf.external_services_manage_your_mulesoft_anypoint_platform_connection.htm&type=5[Salesforce documentation]. 
+
+[[repairing-org-connection]]
+== Repair the Connection to a Salesforce Organization
+
+If the Salesforce *My Domain* is changed in the connected Salesforce organization, you can rediscover the Salesforce organization and repair the connection. 
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+. Click *Salesforce*.
+. Click the *Actions* menu (*...*) in the Salesforce organization to repair the connection for and select *Repair Org Info*.
```

Comment:
I think you can remove the "to repair the connection for" because it's obvious from the context. It would simplify the step.